error_dict = {
    [
        134873102: "Trace message script.manager",
        134873103: "Trace message script.python",
        134873154: "Scheduler message",
        134873248: "Converting the Scheduler program",
        134873344: "Error cleared",
        134873345: "Command 'Errors cleared' executed",
        134873346: "No pending errors and warnings",
        134873347: "Command 'Reset all pending diagnostics' executed",
        134873360: "Trace message common.log.diagnosis",
        134873362: "Trace message common.log.trace",
        134873856: "Security: Change in user management",
        134873857: "Security: Change in certificate management",
        134873858: "Security: Change in firewall configuration",
        134873859: "Security: Change in vpn configuration",
        134873860: "Security: Change to remote logging administration",
        134874112: "Loading configuration started",
        134874113: "Loading configuration successfully finished",
        134874114: "Saving configuration started",
        134874115: "Saving configuration successfully finished",
        134874128: "Trace message",
        134874144: "Creating setup",
        134874145: "Setup created",
        134874146: "Setup started",
        134874147: "Setup finished",
        134874192: "Archive successfully created",
        134874193: "Archive successfully restored",
        134874368: "Change in network management",
        134874384: "PackageManager information",
        134874400: "Storage management information",
        134874416: "System management change",
        134874624: "Trace message comm.datalayer",
        134874625: "Trace message comm.datalayer.framework",
        134874626: "Trace message web.automation",
        134874627: "Trace message common.systemhandler",
        134874628: "Trace message app.automationframework",
        134874704: "Application 'Automation Core' event",
        134877183: "Trace message common.scheduler",
        135135246: "Trace warning script.manager",
        135135247: "Trace warning script.python",
        135135255: "Hardware watchdog warning",
        135135267: "Utilization in the critical section 'WARNING'",
        135135392: "Converting the Scheduler program",
        135135504: "Trace warning common.log.diagnosis",
        135135506: "Trace warning common.log.trace",
        135135744: "HTTP client error",
        135136000: "Security: User management warning",
        135136001: "Security: Certificate management warning",
        135136003: "Security: Warning in the VPN configuration",
        135136004: "Security: Warning in remote logging administration",
        135136005: "Bootloader unlocked!",
        135136272: "Trace warning",
        135136512: "Warning in network management",
        135136528: "PackageManager warning",
        135136544: "Storage management warning",
        135136560: "System management warning",
        135136576: "Warning detected during health monitoring",
        135136592: "License manager warning",
        135136767: "Device Administration: Warning",
        135136768: "Trace warning comm.datalayer",
        135136769: "Trace warning comm.datalayer.framework",
        135136770: "Trace warning web.automation",
        135136771: "Trace warning common.systemhandler",
        135136772: "Trace warning app.automationframework",
        135139327: "Trace warning common.scheduler",
        135200768: "Error in process handling for script instance",
        135200769: "Error during backup or restoration of script manager",
        135200782: "Trace error script.manager",
        135200783: "Trace error script.python",
        135200791: "Task watchdog error",
        135200803: "Error on configuration",
        135200834: "Error in Scheduler system",
        135200870: "Error when restoring the configuration",
        135200904: "Error during automatic reallocation of the callables",
        135200928: "Failed to create a callable",
        135200944: "Condition detected that triggered the safe mode",
        135201024: "Error in diagnostic system",
        135201040: "Trace error common.log.diagnosis",
        135201042: "Trace error common.log.trace",
        135201280: "HTTP server error",
        135201536: "Security: Error in certificate management",
        135201537: "Security: Error in the user management",
        135201539: "Security: Error in the VPN configuration",
        135201540: "Security: Error in remote logging administration",
        135201792: "Loading configuration failed",
        135201794: "Saving configuration failed",
        135201808: "Trace error",
        135201824: "Creating setup failed",
        135201826: "Setup failed",
        135201872: "Backup failed",
        135201873: "Restore operation failed",
        135201874: "Restore operation failed completely",
        135202048: "Error in network management",
        135202064: "PackageManager error",
        135202065: "System management error",
        135202080: "Storage management error",
        135202081: "Memory task processing failed",
        135202096: "System management error",
        135202097: "Error during image upload process",
        135202098: "Factory reset cannot be executed",
        135202099: "Factory reset preparation failed",
        135202112: "Failure detected during health monitoring",
        135202128: "Error in license management",
        135202144: "Error in device discovery",
        135202304: "Trace error comm.datalayer",
        135202305: "Trace error comm.datalayer.framework",
        135202306: "Trace error web.automation",
        135202307: "Trace error common.systemhandler",
        135202308: "Trace error app.automationframework",
        135202384: "Application 'Automation Core' error",
        135204352: "Loading configuration not possible in current state",
        135204863: "Trace error common.scheduler",
        135208960: "Error while running script",
        135208961: "Error in task/requ. comm. between script manager/instance",
        135208962: "Error in Data Layer of script manager",
        135208963: "Error in system configuration",
        135225367: "Task watchdog error",
        135225369: "Utilization in the critical section 'FATAL'",
        135225379: "Utilization in the critical section 'ERROR'",
        135225410: "Error in Scheduler system",
        135225504: "Error in the equidistance of the Scheduler",
        135225520: "Condition detected that triggered a safe machine state",
        135226704: "Critical error in license management",
        135233559: "Hardware watchdog error",
        135233602: "Error in Scheduler system",
        135233712: "Exception occurred",
        135238976: "Critical failure detected during health monitoring",
        135238992: "Fatal error in license management",
        151650304: "Commanded dynamic parameters exceed axis limits",
        151650305: "Commanded dynamic parameters exceed kinematic limits",
        151650306: "Configuration modified booting",
        151650307: "Motion object error level set by application",
        151650308: "Waiting for motion license",
        151650320: "Message in the jog command of the axis",
        151650321: "Message in the jog command of the kinematics",
        151650322: "Message in the axis configuration",
        151650323: "Message in the kinematic configuration",
        151650817: "AxisProfile booting message",
        151651071: "Trace message common.axisprofile",
        151651583: "Trace message plc.cmp.motion",
        151651584: "Axis limits exceeded",
        151651585: "Command message for the synchronized position motion",
        151654397: "Trace message motion.script",
        151654398: "Trace message motion.collisionavoidance",
        151654399: "Trace message motion.core",
        151912448: "Commanded dynamic parameters exceed axis limits",
        151912449: "Commanded dynamic parameters exceed kinematic limits",
        151912450: "Gantry configuration inconsistent",
        151912464: "Warning in jog command of the axis",
        151912465: "Warning in the jog command of the kinematics",
        151912961: "AxisProfile booting warning",
        151913215: "Trace warning common.axisprofile",
        151913727: "Trace warning plc.cmp.motion",
        151913728: "Warning: Axis limits exceeded",
        151916541: "Trace warning motion.script",
        151916542: "Trace warning motion.collisionavoidance",
        151916543: "Trace warning motion.core",
        151977984: "General AxisProfile error",
        151977985: "AxisProfile configuration error",
        151977986: "AxisProfile profile error",
        151977987: "AxisProfile Data Layer provider error",
        151977988: "Switching the operation mode failed",
        151977989: "AxisProfile error on connected device",
        151977990: "Error in axis profile of the touch probe",
        151977991: "Error when scaling AxisProfile",
        151978064: "License for current configuration missing",
        151978065: "Axis configuration error",
        151978066: "Kinematic configuration error",
        151978067: "Error in system configuration",
        151978068: "Error during backup or restoration",
        151978751: "Trace error common.axisprofile",
        151979008: "Error in plc data",
        151979263: "Trace error plc.cmp.motion",
        151979264: "Error when loading the calculation pipeline",
        151979265: "Error when configuring the calculation pipeline",
        151979266: "Non-fatal error when validating the calculation pipeline",
        151979267: "Non-fatal error in axis command 'GearInPos'",
        151979268: "Non-fatal error in calculation pipeline",
        151979269: "Error when calling calculation step by the Data Layer",
        151979270: "Error when calling calculation step param by the Data Layer",
        151979271: "MAIN_DIAG_MOTION_CORE_GEARINVEL_CMD_ERROR_F0",
        151982077: "Trace error motion.script",
        151982078: "Trace error motion.collisionavoidance",
        151982079: "Trace error motion.core",
        151986176: "Device error",
        151986177: "General command error",
        151986178: "General axis command error",
        151986179: "Error in axis command 'AddToKin'",
        151986180: "Error in axis command 'RemoveFromKin'",
        151986181: "Error in axis command 'AddToMaster'",
        151986182: "Axis limit exceeded",
        151986183: "Error in axis jog command",
        151986184: "Error in axis Gantry command",
        151986185: "Error in axis command 'AddToGantry'",
        151986186: "Error in axis command 'RemoveFromGantry'",
        151986187: "Axis configuration error",
        151986188: "General axis command error",
        151986189: "Error in abort command of the axis",
        151986190: "Error in axis reset command",
        151986191: "Error in axis command MoveVelocity",
        151986192: "Kinematic configuration error",
        151986193: "General kinematic command error",
        151986194: "Kinematic limit exceeded",
        151986195: "Error in kinematic jog command",
        151986196: "Error in kinematic raw ipo command",
        151986197: "Error in additional command channel",
        151986198: "General command error",
        151986199: "Override error",
        151986200: "Error in kinematic command 'Group'",
        151986201: "Error in kinematic command 'Move'",
        151986202: "Error in kinematic command 'Contour'",
        151986203: "Error in the geometry object of the kinematics",
        151986204: "Error in the kinematic command 'PreWait'",
        151986208: "Error in system configuration",
        151986209: "Error during backup or restoration",
        151986224: "Error in command options",
        151986225: "Error in command option 'Blending'",
        151986226: "Error in command option product coordinate system",
        151986227: "Error in command option of machine coordinate system",
        151986228: "Error in cmd opt 'SafeZones' (safe zones and working areas)",
        151986229: "Error in command option 'PolyTrans'",
        151986240: "System error",
        151986256: "Error while running script",
        151986257: "Error in task/request comm. between script manager/instance",
        151986258: "Error in the system configuration",
        151986272: "Error in axis command 'EStop'",
        151986273: "Error in axis command 'PosAbs / PosRel / PosAdd'",
        151986274: "Error in axis command 'Power'",
        151986275: "Error in axis command 'Homing'",
        151986276: "Error regarding to the touch probe command of the Motion",
        151987456: "Error in axis command 'GearInPos'",
        151987457: "Error in axis command 'SyncPosGeneric'",
        151987458: "Error in the synchronized Motion command",
        151987459: "Error in the axis command 'gearInVel'",
        151987460: "Error in the axis command 'SyncVelGeneric'",
        152002560: "Error in the Sequence Manager",
        152002561: "Forced stop of the ctrlX Motion",
        152002576: "PLC detected error event",
        152010769: "General error in CXA_Motion"
    ]
}
